### DISCUSSION TITLE
Available PHP versions on Apple silicon

### DISCUSSION THREAD

**Discussion by wturrell** [Homebrew/discussions#4035] [Everyday usage] [answered]
Currently I'm running Catalina 10.15.7, shortly I will upgrade to a Mac with an M2 chip.
Can I get packages for PHP versions prior to 8.2 (essentially 7.4, 8.0, and 8.1)?

(On Catalina I still have the old versions installed and I use link/unlink to switch between them.)

I see from other discussions there is [this tap](https://github.com/shivammathur/homebrew-php), but if there's a "better"/preferred way…

**Comment by SMillerDev** [comment, marked as answer]
You'll be able to install all versions upstream supports (currently 8.0 and up) from Homebrew/core and taps can provide other software too.
